common/src/chain/transaction/account_outpoint.rs

```diff
@@ -110,6 +115,10 @@ pub enum AccountCommand {
     // Change the authority who can authorize operations for a token
     #[codec(index = 5)]
     ChangeTokenAuthority(TokenId, Destination),
+    #[codec(index = 6)]
+    WithdrawOrder(OrderId),
+    #[codec(index = 7)]
+    FillOrder(OrderId, OutputValue, Destination),
```

Comment:
I'm not sure that Destination here make sense but an input is required to have something to sign. For orders it authorise nothing because anyone can fill an order.

Reply:
I think there should be a signature for it anyway to ensure that no one tampers with it.